src/styles/utils/generators/TooltipStyleUtils.ts

```diff
@@ -180,32 +191,40 @@ const createTooltipStyleUtils: StyleUtilGenerator<GetTooltipStylesStyleUtil> = (
         let pointerWrapperTop = 0;
         let pointerWrapperLeft = 0;
         let pointerAdditionalStyle = {};
+        let position: ViewStyle = positioning.pAbsolute;
 
         if (isTooltipSizeReady) {
-            // Determine if the tooltip should display below the wrapped component.
-            // If either a tooltip will try to render within GUTTER_WIDTH or desktop header logical pixels of the top of the screen,
-            // Or the wrapped component is overlapping at top-center with another element
-            // we'll display it beneath its wrapped component rather than above it as usual.
-            shouldShowBelow =
-                shouldForceRenderingBelow ||
-                yOffset - tooltipHeight - POINTER_HEIGHT < GUTTER_WIDTH + titleBarHeight ||
-                !!(tooltip && isOverlappingAtTop(tooltip, xOffset, yOffset, tooltipTargetWidth, tooltipTargetHeight));
+            // Temporarily simplify the logic for native
+            if (isWeb) {
```

Comment:
It's great that we extracted the web-specific part of the styling!

Unfortunately, as you might know, the Expensify engineers are very strongly against using `getPlatform()` and "platform ifs" in any case where it's possible to use platform-specific files (i.e. `index.ts` / `index.native.ts` / ...).

In this case, it **is** possible to use platform-specific files, so we should refactor this code to use them 🤷‍♂️

Reply:
Consistency is important. The rule is there, and I don't think it would be worse when we switch to platform-specific files. So let's just do it.